packages/multi-test/src/app.rs

```diff
@@ -26,7 +26,7 @@ where
     C: Clone + fmt::Debug + PartialEq + JsonSchema,
 {
     // TODO: allow T != Empty
-    pub messages: Vec<CosmosMsg<C>>,
+    pub messages: Vec<SubMsg<C>>,
```

Comment:
This can probably be changed to `CosmosMsg` too. 

Also, the comment above is outdated. We now have support for the custom type `C` and the comment was forgotten in this commit: https://github.com/CosmWasm/cosmwasm-plus/commit/ca59a96d696d535f8692b2d599aa7041677eb0ce